### DISCUSSION TITLE
Update to "primereact": "^10.0.9"

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#374] [PrimeReact] [answered]
I had an issue attempting to update PrimeReact to the latest version.
In run npm i primereact  followed by npm run build 
I get the error message that seems to point to Prime react components
`typeError: Cannot read properties of undefined (reading 'unstyled')
    at isUnstyled (/Users/christopherhunter-johnson/Documents/Dev/SS/ss-next-ui/node_modules/primereact/componentbase/componentbase.cjs.js:185:38)
    at cx (/Users/christopherhunter-johnson/Documents/Dev/SS/ss-next-ui/node_modules/primereact/componentbase/componentbase.cjs.js:190:17)
    at /Users/christopherhunter-johnson/Documents/Dev/SS/ss-next-ui/node_modules/primereact/ripple/ripple.cjs.js:199:33`

I am running NextJs 13.5.4
The package.json script is:
"build": "next build"

This appears to be an issue with 10.0.9.
If I revert to 10.0.8 nextjs build works.
Steps taken to resolve the issue
` npm uninstal primereact
  npm i primereact@10.0.8`

**Comment by melloware** [comment, marked as answer]
Will be fixed in 10.0.10: https://github.com/primefaces/primereact/issues/5168
